scripts/linux/psv/build_psv_auth.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2021 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+mkdir -p build_auth
+cd build_auth
+
+cmake \
+    -DBUILD_ONLY=authentication \
```

Comment:
It would be much better to test the whole Data SDK build without cache instead. Since `OLP_SDK_ENABLE_DEFAULT_CACHE` parameter affects each component.